src/components/InfoPanel/infoPanel.module.scss

```diff
@@ -37,9 +39,11 @@
     width: 150px;
   }
 }
+
 hr {
-  height: 100px;
+  margin: 20px 0;
+  max-height: 100px;
 }
 .params {
-  margin-top: 30px;
+  margin: 0 5px;
```

Comment:
It's okay. But I would do this by padding on the parent level.

Reply:
done